docs/design/Borders.stories.mdx

```diff
@@ -30,12 +30,12 @@ export function Example({ of: size }) {
   return <div style={style} />;
 }
 
-| Width               | Visual                    |
-| :------------------ | :------------------------ |
-| `--border-base`     | <Example of="base" />     |
-| `--border-thick`    | <Example of="thick" />    |
-| `--border-thicker`  | <Example of="thicker" />  |
-| `--border-thickest` | <Example of="thickest" /> |
+| Width               | Visual                    | Value |
+| :------------------ | :------------------------ | :---- |
+| `--border-base`     | <Example of="base" />     | 1 px  |
+| `--border-thick`    | <Example of="thick" />    | 2 px  |
+| `--border-thicker`  | <Example of="thicker" />  | 4 px  |
+| `--border-thickest` | <Example of="thickest" /> | 8 px  |
```

Comment:
```suggestion
export function BorderSize({ of: size }) {
  const element = document.createElement("div");
   element.style.borderBottomStyle = "solid";
  element.style.borderBottomWidth = `var(--border-${size})`;
  document.body.appendChild(element);
  const value = getComputedStyle(element).borderBottomWidth;
  element.remove();
  return value;
}

| Width               | Visual                    | Value |
| :------------------ | :------------------------ | :---- |
| `--border-base`     | <Example of="base" />     | <BorderSize of="base" /> |
| `--border-thick`    | <Example of="thick" />    | <BorderSize of="thick" /> |
| `--border-thicker`  | <Example of="thicker" />  | <BorderSize of="thicker" /> |
| `--border-thickest` | <Example of="thickest" /> | <BorderSize of="thickest" />  |
```

Reply:
@miqdadamirali our designer @chris-at-jobber suggested that the values be calculated dynamically in the docs here in case they change in the future so I added a suggestion to do so. I tested this out locally and it looks good. 
![Screenshot 2024-11-12 at 3 15 12 PM](https://github.com/user-attachments/assets/7b6b705a-1c2f-43ac-a289-9d8164d923f6)

If this looks all good to you please commit the suggestion and then I can merge in the changes.

Reply:
Hey @Aiden-Brine, inspired by your suggestion, i tried something different, if that's good, we can use it, otherwise i'll commit yours and let's merge this :) 
commit: 8898ff1c5c3aafb3abc87377828584d171edb6cd